node/tests/src/main.rs

```diff
@@ -83,6 +83,15 @@ pub async fn sanity_test() {
     }
 }
 
+/// Sanity test for the RPC server.
+/// We use unwraps here because this function is intended to be used like a test.
+pub async fn delay_test() {
```

Comment:
This needs to support the scheduling of the chaos action, meaning to apply & remove it at specific views.

Reply:
We can make request to the node for the its latest view and start the delay after a certain number. Right now the delay starts in the beginning of the test with a set duration of 10 seconds, is it necessary to check for a certain view?

Reply:
Yes, most byzantine behavior duration will need to be "denominated" in views, not clock time, for actually testing liveness during the disruption (and not just quick recovery once it's over). 

It might be needed only for the removal, while applying it always at the beginning of the test, but it's better to just have the test framework supports chaos scheduling with  `from_view`,`to_view` args.